pkg/kafka/topic/validators_test.go

```diff
@@ -1,6 +1,21 @@
 package topic
 
-import "testing"
+import (
+	"testing"
+
+	"github.com/redhat-developer/app-services-cli/pkg/localize"
+	"github.com/redhat-developer/app-services-cli/pkg/localize/goi18n"
+)
+
+var validator *Validator = &Validator{
+	Localizer: getValidator(),
+}
+
+func getValidator() localize.Localizer {
```

Comment:
Is there a better way to initialize the localizer in test files.

Reply:
You can use a `func init() {}` function which should be run before everything else in a Go file.